linux/networking-and-security/ssh-connections/what-is-ssh.md

```diff
@@ -69,19 +79,42 @@ What are the three utilities SSH comprises?
 - rlogin, ssh, rcp
 - rlogin, rsh, rcp
 
-
 ---
 
 ## Footnotes
 
-[1:Public-key encryption]
-It uses a public and a private key to encrypt data. These keys are large numbers that have been paired together. The public key can be shared with everyone, whilst the private key is kept secret.
-
-[2:Diffie–Hellman key exchange]
-It's a method of digital encryption which uses numbers raised to specific powers to generate decryption keys.
-
-[3: AES]
-It stands for Advanced Encryption Standard. It was also chosen by the U.S. government to protect classified data.
-
-[4: Blowfish]
-It's an encryption algorithm that uses an variable-length key (form 32 to 448 bits).
+[1: SSH Private and Public Keys]
+
+In SSH, the public and private keys work such that anyone with a copy of the public key can encrypt data which can then only be read by the person who holds the corresponding private key.
+
+> 💡 Your private key shouldn't be shared with anyone.
+
+The simplest way to generate a key pair is to run `ssh-keygen` without arguments.
+
+Here's an example of the output:
+
+```
+ssh-keygen
+Generating public/private rsa key pair.
+Enter file in which to save the key (/home/enki/.ssh/id_rsa):
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+Your identification has been saved in /home/enki/.ssh/id_rsa.
+Your public key has been saved in /home/enki/.ssh/id_rsa.pub.
+The key fingerprint is:
+SHA256:Up6KjbnEV4Hgfo75YM393QdQsK3Z0aTNBz0DoirrW+c enki@example
+The key's randomart image is:
++---[RSA 2048]----+
+|    .      ..oo..|
+|   . . .  . .o.X.|
+|    . . o.  ..+ B|
+|   .   o.o  .+ ..|
+|    ..o.S   o..  |
+|   . %o=      .  |
+|    @.B...     . |
+|   o.=. o. . .  .|
+|    .oo  E. . .. |
++----[SHA256]-----+
+```
+
+The default key file name for the keys depends on the algorithm. In this case `id_rsa` is used because we're using the default RSA algorithm.
```

Comment:
```suggestion
The default file name for the keys depends on the algorithm. In this case `id_rsa` is used because we're using the default RSA algorithm.
```